src/ngraph/builder/quantization/quantized_linear_matmul.hpp

```diff
@@ -0,0 +1,47 @@
+//*****************************************************************************
+// Copyright 2017-2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/coordinate_diff.hpp"
+#include "ngraph/node.hpp"
+
+namespace ngraph
+{
+    namespace builder
+    {
+        namespace quantization
+        {
+            std::shared_ptr<Node> QuantizedLinearMatmul(std::shared_ptr<Node> input,
```

Comment:
Let's drop this builder in favor of the one below w/ zero points.  The zero point API is the correct API.  Inside that builder we can check all zero points.  If they are constant with value 0 we can call QuantizedDot.  Else we fall back to fp32.